tests/base/test_db.py

```diff
@@ -89,3 +90,29 @@ def test_create_user_db(prepare_create_user):
     password_hash = bcrypt.hashpw(TEST_USERS[0][1].encode(), bcrypt.gensalt())
     create_user_db(user_id, username, password_hash)
     assert is_user_created_db(username)
+
+def test_get_user_id_db(prepare_create_user):
+    """Ensure that a username maps to a user_id correctly. """
+    user_id = uuid.uuid4().bytes
+    username = prepare_create_user
+    password_hash = bcrypt.hashpw(TEST_USERS[0][1].encode(), bcrypt.gensalt())
+    create_user_db(user_id, username, password_hash)
+    assert get_user_id_db(username) == user_id
+
+@pytest.mark.parametrize('video_id, user_id, extension', [
```

Comment:
Change to `@pytest.mark.parametrize('video_id, user_id, extension, success', [`

Reply:
the success I was missing. 💯